.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,53 @@
+name: ciworkflow
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  # Setting an environment variable with the value of a configuration variable
+  ACCOUNT_ID: ${{ secrets.ACCOUNT_ID }}
+  API_KEY: ${{ secrets.API_KEY }}
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 7.x.x
+      - name: Install unzip
+        run: |
+          sudo apt-get update  
+          sudo apt-get install -y unzip
+      - name: Install OpenSSH and Git
+        run: |
+          sudo apt-get update && sudo apt-get install -y openssh-client git
+      - name: siftDotNetIntegrationTest
+        run: |
+          ./build.sh --target=restore
+          ./build.sh --target=generate
+          ./build.sh --target=build
+          ./build.sh --target=test
+          ./build.sh --target=testNet7 --filter=Account
```

Comment:
Only `master` branch runs integration tests. See https://github.com/SiftScience/sift-dotnet/blob/master/.circleci/config.yml#L71-L74.

Reply:
Fixed

Reply:
I am still seeing this job gets triggered in your PR. Instead of using env variable for condition, I would change it like this.
```
siftDotNetIntegrationTest:
  if: github.ref_name == 'master'
  runs-on: ubuntu-latest
  steps:
      - uses: actions/checkout@v4
      - name: Setup .NET
        uses: actions/setup-dotnet@v4
        with:
          dotnet-version: 7.x.x
      - name: siftDotNetIntegrationTest
        run: |
          ./build.sh --target=restore
...
```

Reply:
added `if: ${{ github.ref == 'refs/heads/master' }}`